install.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
```

Comment:
Some stuff about the install script:

- There already exists a very simple script in `scripts/install.sh`, you should probably replace that an move the file there.
- Why do you set the `$CURRENT_HOME` to `$HOME` first and then have it overridden again? Can't you just use `$HOME`?
- I dislike that the home directory is polluted by this script, maybe something like `~/.local/share/discord-screenaudio/git` would be a better place for the repo.

Reply:
> Why do you set the $CURRENT_HOME to $HOME first and then have it overridden again?

Sorry. I copied it from a old script. Thank you for seeing it.

> There already exists a very simple script in scripts/install.sh

I thought of replacing it, but didn't wanted to do that without permission.

> I dislike that the home directory is polluted by this script

True. I am already doing the changes for the installation script.